apps/prairielearn/src/pages/administratorSettings/administratorSettings.html.ts

```diff
@@ -88,6 +91,18 @@ export function AdministratorSettings({ resLocals }) {
               </script>
             </div>
           </div>
+          ${isEnterprise() && (config.openAiApiKey || config.openAiOrganization)
```

Comment:
An organization without an API key won't do us any good.

```suggestion
          ${isEnterprise() && (config.openAiApiKey && config.openAiOrganization)
```